Sources/WalletConnectSign/Auth/Services/Wallet/AuthRequestSubscriber.swift

```diff
@@ -73,7 +73,12 @@ class AuthRequestSubscriber {
     func respondError(payload: SubscriptionPayload, reason: SignReasonCode) {
         Task(priority: .high) {
             do {
-                try await networkingInteractor.respondError(topic: payload.topic, requestId: payload.id, protocolMethod: SessionAuthenticatedProtocolMethod(), reason: reason)
+                try await networkingInteractor.respondError(
```

Comment:
let's mark this func private
```    
private func respondError(payload: SubscriptionPayload, reason: SignReasonCode) {
```
to be sure that's it's not called from outside with this tag

Reply:
Done: ✅ https://github.com/WalletConnect/WalletConnectSwiftV2/pull/1364/commits/e739136d1afff966c403aa4fe8243da95d35b73e